.github/workflows/test.yml

```diff
@@ -5,6 +5,9 @@ on:
     pull_request:
         branches:
             - main
+    push:
+        branches:
+            - main
```

Comment:
Are we pushing to main?  Is this for forks?  Cline shouldn't be pushing directly to main, so i'm not sure this is needed for Cline itself.